CONTRIBUTING.md

```diff
@@ -0,0 +1,117 @@
+** replace `dbt-docs` with your repository name in all docs
```

Comment:
We should probably drop this line, right? I think this line is from the template https://github.com/dbt-labs/dbt-oss-template/blob/er/templates/CONTRIBUTING.md